backend/Wheelingful.API/Program.cs

```diff
@@ -30,12 +31,15 @@
     .AddRoles<IdentityRole>()
     .AddEntityFrameworkStores<WheelingfulDbContext>();
 
-builder.Services.AddCoreServices();
-builder.Services.AddApiServices();
-
 builder.Services.AddOptions();
 builder.Services.AddCoreOptions(builder.Configuration);
 
+builder.Services.AddFluentValidationAutoValidation();
```

Comment:
I saw you added it for the book-author endpoints and you added it here as well. do we need it in 2 places?

Reply:
It was implied to be necessary on the repository page:
![image](https://github.com/TellFruit/wheelingful/assets/98281993/b2e28010-212b-4463-bb14-e4ad04aba82e)
But in fact it is not. Now that you mentioned it, I tried to remove it and the validation was functional.